packages/core/admin/admin/src/hooks/useFetchMarketplaceProviders/index.js

```diff
@@ -1,6 +1,21 @@
 import { useQuery } from 'react-query';
-import { useNotification } from '@strapi/helper-plugin';
-import { fetchMarketplaceProviders } from './utils/api';
+import { getFetchClient, useNotification } from '@strapi/helper-plugin';
+import qs from 'qs';
+
+const MARKETPLACE_API_URL = 'https://market-api.strapi.io';
+const { get } = getFetchClient();
+
+const fetchMarketplaceProviders = async (params = {}) => {
+  const { data } = await get(`${MARKETPLACE_API_URL}/providers`, {
+    params,
+    paramsSerializer: {
+      encode: qs.parse,
+      serialize: qs.stringify,
+    },
+  });
+
+  return data;
+};
```

Comment:
this could all be in `L23` which would mean we can correctly use the `useFetchClient` hook?

Reply:
I think it's much of a muchness, we could use native fetch to be honest?

Reply:
refactored both fetchMarketplacePlugins and fetchMarketplaceProviders using fetch